src/com/comeet/UserService.java

```diff
@@ -173,7 +175,62 @@ private ResponseBuilder buildBearerChallenge(AuthContextException authException)
                         .entity(authException.toString());
         return builder;
     }
-
+    
+    
+    
+    /**
+     * Implementing method for GET /organization/searchCriteria
+     * <p>
+     * Example URL:
+     * http://localhost:8080/pfizer.com/searchCriteria
+     * </p>
+     * @param org domain of organization to search for
+     * @return The list of metro areas and their room list names
+     * @throws ServiceResponseException If the result is not 200 OK.
+     * @throws Exception On an unexpected error.
+    */ 
+    @GET
+    @Path("/{organization}/searchCriteria")
+    @Produces("application/json")
+    public List<BuildingList> getSearchCriteria(
+                    @DefaultValue("") @PathParam("organization") String org) throws Exception {
+    
+        //get the search parameters from the database
+        try {
+            RoomsDao roomsDao = new RoomsDao(null);
+            return roomsDao.getCriteria(org);
+        } catch (Exception ex) {
+            ApiLogger.logger.log(Level.SEVERE, "Error getting search criteria database", ex);
+            throw ex;
+        }      
+    }
+    
+    //TO-DO: Needs to be tested since there is no 
```

Comment:
Basic auth works. Send a request with an `Authorization: Basic` header and our test creds.